pkg/apis/featurestates/v1alpha1/doc.go

```diff
@@ -0,0 +1,4 @@
+// Package v1alpha1 contains API Schema definitions for the cns v1alpha1 API group
+// +k8s:deepcopy-gen=package,register
+// +groupName=cns.vmware.com
+package v1alpha1
```

Comment:
I do not see doc.go and register.go under other APIs in cnsoperator. Can we be consistent?

Reply:
We have `register.go` available in the cnsoperator here - https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/pkg/apis/cnsoperator/register.go

Since this API is not part of CNS Operator, I have added it here - `pkg/apis/featurestates/v1alpha1/register.go`

I have removed `doc.go` file to be consistent.